src/Symfony/Component/Form/Extension/Core/EventListener/FixUrlProtocolListener.php

```diff
@@ -36,7 +36,7 @@ public function onSubmit(FormEvent $event)
     {
         $data = $event->getData();
 
-        if ($this->defaultProtocol && $data && \is_string($data) && !preg_match('~^[\w+.-]+://~', $data)) {
+        if ($this->defaultProtocol && $data && \is_string($data) && !str_contains($data, '/') && !preg_match('~^[\w+.-]+://~', $data)) {
```

Comment:
im not sure, but do we have a test for `domain.com/path`?

Reply:
What about checking instead if the the data starts with a slash?

Reply:
there's another issue actually :sweat_smile: #42329

im not sure for "protocol-relative URLs" we should apply `defaultProtocol` :/ i think it makes sense. esp. since protocol-relative URLs are an anti-pattern.

Reply:
I guess that issue would be fixed by this PR as well if I read the code correctly.

Reply:
correct. I was considering for `//domain` a default protocol should be applied, but it's a new feature also :)